client/starwhale/cli/assistance/cli.py

```diff
@@ -0,0 +1,46 @@
+from typing import Tuple
+
+import click
+
+from .host import CommandRetriever
+from .remote import CommandRunner
+
+
+@click.group("assistance", help="Remote assistance")
+def assistance_cmd() -> None:
+    pass
+
+
+@assistance_cmd.command(
+    "host", help="Run a remote assistance host to accept remote commands"
+)
+@click.option("--broker", help="Broker URL", required=True)
```

Comment:
provide a default broker url for users? for example: `assistance-broker.starwhale.cn`